timelock-api/src/main/conjure/timelock-feedback.yml

```diff
@@ -12,7 +12,8 @@ types:
       EndpointStatistics:
         fields:
           p99: double
-          oneMin: double
+          minRate: double
+          errorRate: double
```

Comment:
Unfortunately, this breaks compatibility.
- For backward compatibility (old clients can talk to new servers), the minRate and errorRate need to be optional
- For forward compatibility (new clients can talk to old servers), that isn't readily possible in Conjure so you'll want to add a product dependency. Servers are required to reject unknown fields.

It's not ideal but I'd suggest leaving `oneMin` as it was named